### ISSUE TITLE
Landslide icon?

### ISSUE THREAD

**Issue by robmaggidev** [erikflowers/weather-icons#118]
Hi, I'm working on a weather/emergency-alert website and would like to ask you if you're planning to release an icon for landslide, something like this: https://thenounproject.com/term/landslide/4241/.
Thanks,
Rob

**Comment by erikflowers** [comment]
Yes this can go into the natural disaster category. Mudslide as well? Are those even different?

And Avalanche probably too.
